posts/01-first-map/nquarder/cedar-rapids.md

```diff
@@ -0,0 +1,7 @@
+## Kirkwood Community College (Cedar Rapids, IA)
+
+I went to Kirkwood CC in CR, IA 1999-2001. On July 1, 1966, Kirkwood was officially established, originally called "Area X Community College." The college immediately assumed responsibility for the federally-funded vocational/technical programs the Cedar Rapids Community School District had provided since 1964. [Source](https://creditcatalog.kirkwood.edu/kirkwood-history-accreditation/#:~:text=On%20July%201%2C%201966%2C%20Kirkwood,District%20had%20provided%20since%201964.)
```

Comment:
Maybe change spell out CC and CR into Community College and Cedar Rapids respectively?  Other than that look great!